jsconfig.json

```
@@ -0,0 +1,16 @@
+{
+  "compilerOptions": {
+    "baseUrl": "./src",
+    "paths": {
+      "@apis/*": ["apis/*"],
+      "@assets/*": ["assets/*"],
+      "@components/*": ["components/*"],
+      "@constants/*": ["constants/*"],
+      "@hooks/*": ["hooks/*"],
+      "@pages/*": ["pages/*"],
+      "@stores/*": ["stores/*"],
+      "@utils/*": ["utils/*"],
+      "@types/*": ["types/*"]
+    }
```

절대 경로 설정하실 때 아래처럼 작성하시면 더 간단하게 설정하실 수 있습니다!
```suggestion
    "baseUrl": "./src",
    "paths": {
      "@*": ["*"]
    }
```

---

오 ..!! 감사합니다 !!